code/API_definitions/device-swap.yaml

```diff
@@ -0,0 +1,399 @@
+openapi: 3.0.3
+info:
+  title: Device Swap
+  description: |
+      This API allows to check the last time that the phone (device), where a mobile number (MSISDN) is installed, has changed.
```

Comment:
not sure about "a mobile number (MSISDN) is installed" meaning. What about something simpler as "This API allows to check the last time that the phone (device) - phone number association has changed"

Reply:
modified description